### ISSUE TITLE
Bug: Windows: 4/9 domains have network errors during provisioning

### ISSUE THREAD

**Issue by jcdevnj** [Varying-Vagrant-Vagrants/VVV#2745]
## What was The Command Used To Provision

```shell
vagrant up
```

## What Kind of VVV Provision Was This

This was a fresh install

## Logs/What Broke

During the initial provision, some of the domains cannot be connected to, so there are missing files. I can connect to these domains in my browser, it is only in the vvv provisioner that it fails.

```
shorelineJustin@JCpavillion MINGW64 ~/vvv-local (develop)
$ vagrant up --provision
__ __ __ __
\ V\ V\ V / v3.14 Ruby:3.3.6, Path:"C:/Users/shorelineJustin/vvv-local"
 \_/\_/\_/  git::develop(dbd10713)

Copying config/default-config.yml to config/config.yml
IMPORTANT NOTE: Make all modifications to config/config.yml in future so that they are not lost when VVV updates.

Platform: mingw32 windows  missingWinAdminPriv  vagrant-goodhosts monochrome-terminal shared_db_folder_disabled
Vagrant: v2.4.3, virtualbox: v7.1.4

Docs:       https://varyingvagrantvagrants.org/
Contribute: https://github.com/varying-vagrant-vagrants/vvv
Dashboard:  http://vvv.test

Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'bento/ubuntu-20.04'...
==> default: Matching MAC address for NAT networking...
==> default: Setting the name of the VM: vvv-local_ca904e46c51
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection reset. Retrying...
    default: Warning: Connection aborted. Retrying...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default:
    default: Guest Additions Version: 7.0.18
    default: VirtualBox Version: 7.1
==> default: [vagrant-goodhosts] - found entry for: 192.168.56.4 vvv.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.56.4 one.wordpress.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.56.4 two.wordpress.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.56.4 vvv.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.56.4 one.wordpress.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.56.4 two.wordpress.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.56.4 vvv.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.56.4 one.wordpress.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.56.4 two.wordpress.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.56.4 vvv.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.56.4 one.wordpress.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.56.4 two.wordpress.test
==> default: [vagrant-goodhosts] Checking for host entries
==> default: [vagrant-goodhosts] Finished processing
==> default: Setting hostname...
==> default: Configuring and enabling network interfaces...
==> default: Mounting shared folders...
    default: C:/Users/shorelineJustin/vvv-local/www => /srv/www
    default: C:/Users/shorelineJustin/vvv-local/config => /srv/config
    default: C:/Users/shorelineJustin/vvv-local/log/php => /var/log/php
    default: C:/Users/shorelineJustin/vvv-local/database/sql => /srv/database
    default: C:/Users/shorelineJustin/vvv-local/provision => /srv/provision
    default: C:/Users/shorelineJustin/vvv-local/log/nginx => /var/log/nginx
    default: C:/Users/shorelineJustin/vvv-local/certificates => /srv/certificates
    default: C:/Users/shorelineJustin/vvv-local/log/memcached => /var/log/memcached
    default: C:/Users/shorelineJustin/vvv-local/log/provisioners => /var/log/provisioners
==> default: Detected mount owner ID within mount options. (uid: 0 guestpath: /var/log/memcached)
==> default: Detected mount group ID within mount options. (gid: 0 guestpath: /var/log/memcached)
==> default: Detected mount owner ID within mount options. (uid: 0 guestpath: /var/log/nginx)
==> default: Detected mount group ID within mount options. (gid: 0 guestpath: /var/log/nginx)
==> default: Detected mount owner ID within mount options. (uid: 0 guestpath: /var/log/php)
==> default: Detected mount group ID within mount options. (gid: 0 guestpath: /var/log/php)
==> default: Detected mount owner ID within mount options. (uid: 0 guestpath: /var/log/provisioners)
==> default: Detected mount group ID within mount options. (gid: 0 guestpath: /var/log/provisioners)
==> default: Detected mount owner ID within mount options. (uid: 1000 guestpath: /srv/www)
==> default: Detected mount group ID within mount options. (gid: 33 guestpath: /srv/www)
==> default: Running provisioner: file...
    default: C:/Users/shorelineJustin/vvv-local/version => /home/vagrant/version
==> default: Running provisioner: pre-provision-script (shell)...
    default: Running: inline script
    default:
    default:     ▄▀▀▀▄▄▄▄▄▄▄▀▀▀▄    ▄   ▄    A full provision will take a bit.
    default:     █▒▒░░░░░░░░░▒▒█   █   █     Sit back, relax, and have some tea.
    default:      █░░█░░░░░█░░█   ▀   ▀
    default:   ▄▄  █░░░▀█▀░░░█   █▀▀▀▀▀▀█    If you didn't want to provision you can
    default:  █░░█ ▀▄░░░░░░░▄▀▄▀▀█      █    turn VVV on with 'vagrant up'.
    default: ───────────────────────────────────────────────────────────────────────
    default:
==> default: Running provisioner: default (shell)...
    default: Running: C:/Users/SHOREL~2/AppData/Local/Temp/vagrant-shell20241113-6128-tvevgo.sh
    default:  - skipping ntpdate clock sync, not installed yet
    default:  ▷ Running the 'main' provisioner...
    default:  ▷ Running init hook
    default:  * Bash profile setup and directories.
    default:  * Reloading SSH Daemon
    default:  * checking Ubuntu version
    default:  * Copying /srv/provision/core/vvv/apt-conf-d/99hashmismatch to /etc/apt/apt.conf.d/99hashmismatch
    default:  ✔ Finished init hook in 0m 0s 590ms
    default:  * Testing network connection to https://ppa.launchpadcontent.net with wget -q --spider --timeout=5 --tries=3 https://ppa.launchpadcontent.net
    default:  * Successful Network connection to https://ppa.launchpadcontent.net detected
    default:  * Testing network connection to https://wordpress.org with wget -q --spider --timeout=5 --tries=3 https://wordpress.org
    default:  ! Network connection issues found. Unable to reach https://wordpress.org
    default: Spider mode enabled. Check if remote file exists.
    default: --2024-11-13 22:02:28--  https://wordpress.org/
    default: Resolving wordpress.org (wordpress.org)... failed: Connection timed out.
    default: wget: unable to resolve host address ‘wordpress.org’
    default:  * Testing network connection to https://github.com with wget -q --spider --timeout=5 --tries=3 https://github.com
    default:  ! Network connection issues found. Unable to reach https://github.com
    default: Spider mode enabled. Check if remote file exists.
    default: --2024-11-13 22:02:38--  https://github.com/
    default: Resolving github.com (github.com)... failed: Connection timed out.
    default: wget: unable to resolve host address ‘github.com’
    default:  * Testing network connection to https://raw.githubusercontent.com with wget -q --spider --timeout=5 --tries=3 https://raw.githubusercontent.com
    default:  ! Network connection issues found. Unable to reach https://raw.githubusercontent.com
    default: Spider mode enabled. Check if remote file exists.
    default: --2024-11-13 22:02:48--  https://raw.githubusercontent.com/
    default: Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.111.133, 185.199.110.133, ...
    default: Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.
    default: HTTP request sent, awaiting response... 301 Moved Permanently
    default: Location: https://github.com/ [following]
    default: Spider mode enabled. Check if remote file exists.
    default: --2024-11-13 22:02:48--  https://github.com/
    default: Resolving github.com (github.com)... failed: Connection timed out.
    default: wget: unable to resolve host address ‘github.com’
    default:  * Testing network connection to https://getcomposer.org with wget -q --spider --timeout=5 --tries=3 https://getcomposer.org
    default:  * Successful Network connection to https://getcomposer.org detected
    default:  * Testing network connection to https://packagist.org with wget -q --spider --timeout=5 --tries=3 https://packagist.org
    default:  * Successful Network connection to https://packagist.org detected
    default:  * Testing network connection to http://mariadb.mirrors.ovh.net with wget -q --spider --timeout=5 --tries=3 http://mariadb.mirrors.ovh.net
    default:  ! Network connection issues found. Unable to reach http://mariadb.mirrors.ovh.net
    default: Spider mode enabled. Check if remote file exists.
    default: --2024-11-13 22:02:58--  http://mariadb.mirrors.ovh.net/
    default: Resolving mariadb.mirrors.ovh.net (mariadb.mirrors.ovh.net)... failed: Connection timed out.
    default: wget: unable to resolve host address ‘mariadb.mirrors.ovh.net’
    default:  * Testing network connection to http://ports.ubuntu.com/ with wget -q --spider --timeout=5 --tries=3 http://ports.ubuntu.com/
    default:  * Successful Network connection to http://ports.ubuntu.com/ detected
    default:  * Testing network connection to https://nginx.org/packages/mainline/ with wget -q --spider --timeout=5 --tries=3 https://nginx.org/packages/mainline/
    default:  * Successful Network connection to https://nginx.org/packages/mainline/ detected
    default: #################################################################
    default:
    default: ! Warning! Network Problems:
    default:
    default: VVV tried to check several domains it needs for provisioning but 4 of 9 failed:
    default:
    default:  [x] https://wordpress.org|
    default:  [x] https://github.com|
    default:  [x] https://raw.githubusercontent.com|
    default:  [x] http://mariadb.mirrors.ovh.net|
    default:
    default: Make sure you have a working internet connection, that you
    default: restarted after installing VirtualBox/Parallels/Vagrant, and that
    default: they aren't blocked by a firewall or security software.
    default: If you can load the address in your browser, then VVV should
    default: be able to connect.
    default:
    default: Also note that some users have reported issues when combined
    default: with VPNs, disable your VPN and reprovision to see if this is
    default: the cause.
    default:
    default: Additionally, if you're at a contributor day event, be kind,
    default: provisioning involves downloading things, a full provision may
    default: ruin the wifi for everybody else :(
    default:
    default: Try provisioning again once network connectivity is restored.
    default: If that doesn't work, and you're sure you have no VPNs and a strong
    default: internet connection, open an issue on GitHub, and include the
    default: output above so that the problem can be debugged
    default:
    default: vagrant halt
    default: vagrant up --provision
    default:
    default: https://github.com/Varying-Vagrant-Vagrants/VVV/issues
    default:
    default: #################################################################
    default:  ==================================================================================================
    default:  ! If this check fails despite succeeding in the browser, contact us in Slack or GitHub immediately
    default:  ==================================================================================================
```

`config.yml` is the default one from this repo.

## Steps to Reproduce (for bugs)
1. Follow the default instructions for installing VVV

[provisioner-main.log](https://gist.github.com/jcdevnj/2675905450aceee731f51632d3b4f82a)

## Your Environment

```
OS Name	Microsoft Windows 11 Home
Version	10.0.22631 Build 22631
```

```
__ __ __ __
\ V\ V\ V / v3.14 Ruby:3.3.6, Path:"C:/Users/shorelineJustin/vvv-local"
 \_/\_/\_/  git::develop(dbd10713)

Platform: mingw32 windows  missingWinAdminPriv  vagrant-goodhosts monochrome-terminal shared_db_folder_disabled
Vagrant: v2.4.3, virtualbox: v7.1.4
```

**Comment by tomjn** [comment]
Did you just install or upgrade virtualbox and forgot to do a full restart?
I wouldn’t expect failed network checks to stop provisioning but they
shouldn’t have failed.

Usually the best way to reproduce this is to install virtualbox and not
reboot so the virtualbox network drivers don’t get a chance to do their
thing

Likewise check the hosts file for the entries for the domains you expected,
as well as for any duplicates

**Comment by tomjn** [comment]
Do you have a VPN/behind university internet per chance?

**Comment by jcdevnj** [comment]
I restarted after VirtulBox, and checked the hosts file, but I think you're on to something with the internet being the issue. On my home network, using the same laptop, it connects fine. At the office, those 4 domains fail to connect. I'm not behind a VPN and it's not university internet though.

**Comment by jcdevnj** [comment]
Downgrading to VirtualBox `7.0.18` resolved the issue for me. On the newest VirtualBox `7.1.4`, it takes about 8 seconds to resolve the DNS for wordpress.org and the `wget` command in the main provisioner times out at around 5 seconds. For some reason, the DNS resolves much faster in `7.0.18` and the provisioner doesn't time out. But again, it only times out when using a specific WiFi network.

Thanks for your help!
